docs/override_check.py

```diff
@@ -0,0 +1,43 @@
+#!/usr/lib/env python
+#
+# Module Documentation Override Check
+#
+# Verifies that any modules present in the _templates/overrides directory have all of their
+# their exported functions included in the doc file.
+
+from __future__ import print_function
+
+import glob
+import importlib
+import os
+import sys
+
+
+failed = False
+for full_path in glob.glob('_templates/overrides/metpy.*.rst'):
+
+    filename = os.path.basename(full_path)
+    module = filename.split('.rst')[0]
+
+    # Get all functions in the module
+    i = importlib.import_module(module)
+    functions = set(i.__all__)
```

Comment:
Just curious: is using `set()` here something that came out of an issue you ran into? Or just being defensive?

Reply:
Not out of any particular issue, but I did it that way because I thought of the problem as a set difference (are there any functions in the set of functions in the module that are not in the set of functions listed on the page), so I made both `functions` and `lines` sets.